wp-content/themes/pub/wporg-learn-2020/css/components/_section.scss

```diff
@@ -1,4 +1,11 @@
 .section-heading {
+	&.row {
+
+		@media only screen and (min-width: $breakpoint-tablet) {
+			align-items: baseline;
```

Comment:
Corrects the alignment when the elements have different size fonts, eg. H1 and link text which is paragraph size